web/static/js/consent.js

```diff
@@ -0,0 +1,48 @@
+module.exports = (function() {
```

Comment:
You're right, there's no point exporting the function, I was just experimenting with how Phoenix/brunch compiled the js.

 `import` vs `require` and `export default` vs `module.exports` is literally just ES5 vs ES6, and I was just doing what I'm used to.

I've pushed up some updates, I think you'll need some environment variables to test it